src/features/evaluation-results/[id]/view-evaluation-results-footer.tsx

```diff
@@ -1,12 +1,18 @@
 import { useNavigate } from "react-router-dom"
 import { Button } from "../../../components/ui/button/button"
 import { Icon } from "../../../components/ui/icon/icon"
+import { useAppSelector } from "../../../hooks/useAppSelector"
 
 export const ViewEvaluationResultsFooter = () => {
   const navigate = useNavigate()
+  const { previousUrl } = useAppSelector((state) => state.evaluationAdministrations)
```

Comment:
This is for evaluation administrations only. Should we move this to the app slice?